### PR TITLE
[PM-14964] revert passphrase minimum

### PR DESCRIPTION
**Pull request by audreyality**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-14964

## 📔 Objective

Revert 0254550b07277be7f0d6fedcd862b462e80947a3 and explain recommended boundaries using hint text.

## 📸 Screenshots

No policy; value exceeds recommended value:

image

image

No policy; value less than or equal to recommended value:

image

image

With policy; value exceeds recommended value:

image

image

With policy' value less than or equal to recommended value:

image

image

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Comment by ginat** [comment]
Minimum values should not be changed for password or passphrase generator below 5 characters and 3 words, respectively. Did that mean to happen?

**Comment by audreyality** [comment]
No. I haven't updated my tests yet, so there's bugs. :/

**Comment by ginat** [comment]
All good, thank you for getting the PR party started!

**Comment by audreyality** [comment]
Hi @bwbug. This PR is a hotfix whose scope is limited to reverting the minimum to its prior value. We've gotten some internal feedback about the hint text; that'll be addressed before this ticket merges.

In the meantime, I'm going to mark your comment off-topic so that the PR can focus on code review feedback. Please continue contributing your feedback concerning more substantial revisions using the [community forums thread](https://community.bitwarden.com/t/avoid-arbitrary-length-restrictions-in-generator/75764).

**Review by vincentsalucci — DISMISSED** [review]
Changes to AC owned file `password-generator.component` look good 👍

**Review by djsmith85 — CHANGES_REQUESTED** [review]
Only one thing regarding the hint and the spacing. The rest is looking great!

File: apps/browser/src/_locales/en/messages.json [hunk 1]
```diff
@@ -2903,6 +2903,26 @@
       }
     }
   },
+  "passwordLengthRecommendationHint": {
+    "message": " Use $RECOMMENDED$ characters or more to generate a strong password.",
```
[the remark this answers is not in the document]
Comment (djsmith85) [reply apps/browser/src/_locales/en/messages.json:2907]: Discussed this offline and is addressed with [9828f29](https://github.com/bitwarden/clients/pull/12019/commits/9828f296d4fe3b6502e3bbeaeaaafd372cdb70bc)

**Review by djsmith85 — APPROVED** [review]

**Review by vincentsalucci — APPROVED** [review]

**Pull request merged**